vue-custom-bootstrap/src/components/PageLayout.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-9 col-8">
+        <slot name="main"></slot>
+        </div>
+        <div class="col-sm-3 col-4">
+        <slot name="right-sidebar"></slot>
+        </div>
+      </div>
+    </div>
+</template>
+
+<script>
+
+export default {
+  name: 'PageLayout',
+  props: {
+    pageId: String
```

Comment:
Is pageId getting used anywhere?

Reply:
Not being used right now so you can remove pageId. Thinking PageLayout would be the main template for now, so it can hold other other content represented by pageId.